selenium-standalone.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   drivers: {
     chrome: {
       // This version needs to match the chrome version on GitHub Actions
-      version: '94.0.4606.41',
+      version: '96.0.4664.45',
```

Comment:
probably best to rebase instead of merging to get rid of this

Reply:
done